Python/Product/TestAdapter.Executor/Pytest/PytestDiscoveryResults.cs

```diff
@@ -50,6 +51,10 @@ sealed class PytestTest {
 
         [JsonProperty("parentid")]
         public string Parentid { get; set; }
+
+        public override string ToString() {
+            return String.Format("{0}: Id:{1} Source:{2}", this.GetType().Name, this.Id, this.Source);
+        }
```

Comment:
i left out some attributes.. maybe is better to just print everything to help tracking down bugs